boards/UNMAINTAINED_kgpe-d16_server-whiptail/UNMAINTAINED_kgpe-d16_server-whiptail.config

```diff
@@ -73,5 +73,5 @@ export CONFIG_BOOT_KERNEL_REMOVE="plymouth.ignore-serial-consoles"
 export CONFIG_BOOT_DEV="/dev/sda1"
 export CONFIG_BOARD_NAME="KGPE-D16 Server-whiptail"
 export CONFIG_USB_BOOT_DEV="/dev/sdb1"
-export CONFIG_FLASHROM_OPTIONS="--force --noverify -p internal"
+export CONFIG_FLASH_OPTIONS="flashprog memory --progress --programmer internal"
```

Comment:
Probably better to omit the `memory` for now. I want to keep things backwards compatible anyway and the particular syntax is not reviewed yet.

Reply:
@i-c-o-n changed under [668f697](https://github.com/linuxboot/heads/pull/1769/commits/668f6974645a11a5e47c02c07223b634d47289e8)

see https://github.com/linuxboot/heads/compare/dd936d8d36111b7b905847fb6be98fbbe2e51cf8..668f6974645a11a5e47c02c07223b634d47289e8